### ISSUE TITLE
Import recording fails

### ISSUE THREAD

**Issue by fthakur** [blindsidenetworks/scalelite#642]
published_status = if recording.publish_updated is causing import failure. Failed to import recording: undefined method `publish_updated'.
This was introduced in version 1.1.7

**Comment by amalrails** [comment]
@fthakur did you run docker exec -t scalelite-api bundle exec rake db:migrate mentioned in the release notes

**Comment by fthakur** [comment]
Yes I did. It faults because it was run earlier and callback_data is already added to the database. This results in publish_updated not getting added to the database. There should be a check if the fields exist.

**Comment by amalrails** [comment]
@fthakur can do you a docker exec -i scalelite-api bundle exec rake db:rollback ? and try again docker exec -t scalelite-api bundle exec rake db:migrate

**Comment by fthakur** [comment]
I get an error:
ActiveRecord::UnknownMigrationVersionError:
No migration with version number 20181123180008.

This is the error that i get when running migrate:
StandardError: An error has occurred, this and all later migrations canceled:
PG::DuplicateTable: ERROR:  relation "callback_data" already exists

**Comment by eigernick13** [comment]
are this bug already fix?

**Comment by jetchevers** [comment]
Hello, 
I have the same problem, Is there any solution?
Thank you.

**Comment by amalrails** [comment]
> I get an error:
> ActiveRecord::UnknownMigrationVersionError:
> No migration with version number 20181123180008.
> 
> This is the error that i get when running migrate:
> StandardError: An error has occurred, this and all later migrations canceled:
> PG::DuplicateTable: ERROR: relation "callback_data" already exists

i dont understand this error how can you have migration file from 2018?

**Comment by amalrails** [comment]
@eigernick13 @jetchevers are you guys also having ActiveRecord::UnknownMigrationVersionError:
No migration with version number 20181123180008 error or Failed to import recording: undefined method `publish_updated' error?

**Comment by defnull** [comment]
The migration error would likely be fixed with #619

**Comment by jfederico** [comment]
@defnull we are going to merge that PR but change the date to the project inception date. The 2018 is a little out of time :)

**Comment by defnull** [comment]
The date was intentional because it matched the corresponding schema version. https://github.com/blindsidenetworks/scalelite/blob/6b175a4ec5b8c318d8c52256bc8590d9a82836a2/db/schema.rb#L13

I am not sure if changing the date may result in problems again for people with old schema versions trying to migrate.

**Comment by defnull** [comment]
I just checked. Older instances of Scalelite report `20181123180008` as their schema version. Since you changed the version of the migration step in my PR, this step will be applied again, which will break because the tables are already there. My original PR did work as expected. Please revert your changes?

**Comment by jetchevers** [comment]
Hi

I have following error:

**E, [2021-09-11T17:56:50.833066 #6] ERROR -- : Failed to import recording: undefined method `publish_updated' for #<Recording:0x00007fc0bd8e6bd0>**

I do not have error 
ActiveRecord::UnknownMigrationVersionError: No migration with version number 20181123180008

Thanks for your help.

**Comment by jetchevers** [comment]
I ran the command **"docker exec -t scalelite-api bundle exec rake db:migrate"** and the problem was solved
Thanks.

**Comment by chehwi** [comment]
Hi

we have a slightly different error while importing recordings to scalelite from our BBB-Servers. We have setup NFS-Shares between all servers. The error shows after upgrading to Scalelite v1.1.7.1
We have done the db:migrate and we are on schema version 20210725200414

**Now, the importer-service is running an endless loop for the recording! Starts over and over again for the same tar file without any error message, see log below**

```
conf docker[25199]: I, [2021-09-17T11:43:53.766979 #6]  INFO -- : Found metadata file: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/metadata.xml
Sep 17 13:43:53 conf docker[25199]: I, [2021-09-17T11:43:53.806975 #6]  INFO -- : Importing recording from file: /var/bigbluebutton/spool/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570.tar
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-13.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-14.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-1.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-3.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-11.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-4.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-10.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-9.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-4.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-15.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-14.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-5.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-3.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-10.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-13.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-7.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-8.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-1.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-2.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-12.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-6.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-11.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/thumbnails/thumb-9.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-8.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-12.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-5.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-2.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-6.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-7.png
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-4.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-14.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-11.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-7.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-3.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-8.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-2.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-9.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-1.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-10.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-13.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-15.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-6.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-12.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/textfiles/slide-5.txt
Sep 17 13:43:53 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1631878932591/slide-15.png
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation/logo.png
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/deskshare.xml
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/video/
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/video/webcams.webm
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/shapes.svg
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/slides_new.xml
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/metadata.xml
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/panzooms.xml
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/captions.json
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/presentation_text.json
Sep 17 13:43:54 conf docker[25199]: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/cursor.xml
Sep 17 13:43:54 conf docker[25199]: I, [2021-09-17T11:43:54.005952 #6]  INFO -- : Found metadata file: presentation/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570/metadata.xml
Sep 17 13:43:54 conf docker[25199]: I, [2021-09-17T11:43:54.045272 #6]  INFO -- : Importing recording from file: /var/bigbluebutton/spool/5ff1d26062e5b2fa5162b52499e7e2bc3ee153ef-1631878932570.tar
```

**Comment by alizamanir** [comment]
Hello

I have the same issue,
my scalelite version is v1.1.8

the "docker container logs scalelite-recording-importer" returns:

.
.
.
E, [2021-10-05T17:42:50.553257 #8] ERROR -- : Failed to import recording: undefined method `publish_updated' for #<Recording:0x00007f878e8a7498>
Did you mean?  published
I, [2021-10-05T17:42:50.553333 #8]  INFO -- : Importing recording from file: /var/bigbluebutton/spool/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807.tar
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/shapes.svg
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/video/
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/video/webcams.webm
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation_text.json
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/metadata.xml
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/panzooms.xml
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/slides_new.xml
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/deskshare.xml
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/logo.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-8.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-1.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-6.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-10.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-9.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-5.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-3.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-7.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-8.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-4.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/thumbnails/thumb-2.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-4.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-6.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-9.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-2.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-7.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-5.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-3.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-10.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/slide-1.png
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-1.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-6.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-10.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-2.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-5.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-4.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-7.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-9.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-3.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1632117983825/textfiles/slide-8.txt
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/cursor.xml
presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/captions.json
I, [2021-10-05T17:42:50.942862 #8]  INFO -- : Found metadata file: presentation/fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807/metadata.xml
D, [2021-10-05T17:42:50.945074 #8] DEBUG -- :    (0.3ms)  BEGIN
D, [2021-10-05T17:42:50.945692 #8] DEBUG -- :   Recording Load (0.3ms)  SELECT "recordings".* FROM "recordings" WHERE "recordings"."record_id" = $1 LIMIT $2  [["record_id", "fb58002e3a9e322562a55dfb331e5e31d0f32b28-1632117983807"], ["LIMIT", 1]]
D, [2021-10-05T17:42:50.947136 #8] DEBUG -- : #<Recording id: 370, record_id: "fb58002e3a9e322562a55dfb331e5e31d0f32b28-163211798...", meeting_id: "room-126279", name: "14000629-ancd-126279", published: true, participants: 1, state: "published", starttime: "2021-09-20 06:06:23", endtime: "2021-09-20 06:10:58", deleted_at: nil>
D, [2021-10-05T17:42:50.948569 #8] DEBUG -- :   PlaybackFormat Load (0.2ms)  SELECT "playback_formats".* FROM "playback_formats" WHERE "playback_formats"."recording_id" = $1 AND "playback_formats"."format" = $2 ORDER BY "playback_formats"."format" ASC LIMIT $3  [["recording_id", 370], ["format", "presentation"], ["LIMIT", 1]]
D, [2021-10-05T17:42:50.949346 #8] DEBUG -- : #<PlaybackFormat id: 370, recording_id: 370, format: "presentation", url: "/playback/presentation/2.3/fb58002e3a9e322562a55df...", length: 3, processing_time: nil>
D, [2021-10-05T17:42:50.950685 #8] DEBUG -- :   Thumbnail Load (0.3ms)  SELECT "thumbnails".* FROM "thumbnails" WHERE "thumbnails"."playback_format_id" = $1 AND "thumbnails"."sequence" = $2 ORDER BY "thumbnails"."sequence" ASC LIMIT $3  [["playback_format_id", 370], ["sequence", 0], ["LIMIT", 1]]
D, [2021-10-05T17:42:50.952184 #8] DEBUG -- :   Thumbnail Load (0.2ms)  SELECT "thumbnails".* FROM "thumbnails" WHERE "thumbnails"."playback_format_id" = $1 AND "thumbnails"."sequence" = $2 ORDER BY "thumbnails"."sequence" ASC LIMIT $3  [["playback_format_id", 370], ["sequence", 1], ["LIMIT", 1]]
D, [2021-10-05T17:42:50.953651 #8] DEBUG -- :   Thumbnail Load (0.3ms)  SELECT "thumbnails".* FROM "thumbnails" WHERE "thumbnails"."playback_format_id" = $1 AND "thumbnails"."sequence" = $2 ORDER BY "thumbnails"."sequence" ASC LIMIT $3  [["playback_format_id", 370], ["sequence", 2], ["LIMIT", 1]]
D, [2021-10-05T17:42:50.955282 #8] DEBUG -- :   Thumbnail Load (0.3ms)  SELECT "thumbnails".* FROM "thumbnails" WHERE "thumbnails"."playback_format_id" = $1 ORDER BY "thumbnails"."sequence" ASC  [["playback_format_id", 370]]
D, [2021-10-05T17:42:50.956079 #8] DEBUG -- :    (0.2ms)  COMMIT
E, [2021-10-05T17:42:50.982860 #8] ERROR -- : Failed to import recording: undefined method `publish_updated' for #<Recording:0x00007f878e827720>
Did you mean?  published
D, [2021-10-05T17:42:50.982928 #8] DEBUG -- : Starting file monitor…


I tried  the "ocker exec -t scalelite-api bundle exec rake db:migrate" many times bit the same issue occurs.

any one have suggestion on this?

Thanks
